modules/app-configuration/configuration-store/.test/common/main.test.bicep

```diff
@@ -103,9 +103,8 @@ module testDeployment '../../main.bicep' = {
       }
     ]
     softDeleteRetentionInDays: 1
-    systemAssignedIdentity: false
-    userAssignedIdentities: {
-      '${nestedDependencies.outputs.managedIdentityResourceId}': {}
+    managedIdentities: {
```

Comment:
Is this flip intended? Before it had User-assigned identities

Reply:
Thanks, it was not intended, I switched systemAssigned back to false.

Reply:
@AlexanderSehr just found out that the resource supports both identity types at the same time. I will switch systemAssigned to back to "true" if you don't mind